api/lib/FSClient.js

```diff
@@ -0,0 +1,63 @@
+const BoxSDK = require('box-node-sdk');
+const fs = require('fs');
+const path = require('path');
+
+function FSClient() {
+    // Read and parse the automatically created Box configuration file.
+    let configFile = fs.readFileSync(path.dirname(require.main.filename)+'/config.json');
+    configFile = JSON.parse(configFile);
+
+    // Initialize the SDK with the Box configuration file and create a client that uses the Service Account.
+    var sdk = new BoxSDK({
+        clientID: configFile.boxAppSettings.clientID,
+        clientSecret: configFile.boxAppSettings.clientSecret,
+        appAuth: {
+            keyID: configFile.boxAppSettings.appAuth.publicKeyID,
+            privateKey: configFile.boxAppSettings.appAuth.privateKey,
+            passphrase: configFile.boxAppSettings.appAuth.passphrase
+        }
+    });
+
+    // Get an app user client
+    this._client = sdk.getAppAuthClient('enterprise', configFile.enterpriseID);
+}
+
+
+FSClient.prototype.uploadMenu = function(file) {
+    var stream = fs.createReadStream(file.path);
+    return this.getFolderId('TOP - The OCR Project')
```

Comment:
@gabrieldrs I think that would be nice to add a variable in the .env for the root folder name of our Box to make it easy to people that is going to use our repos and also in the README redirect them to Box documentation that you made to generate the config.json

Let me know your thoughts about it! Thanks :smile: